ph5/utilities/segd2ph5.py

```diff
@@ -16,9 +16,11 @@
 import re
 from math import modf
 from ph5.core import experiment, columns, segdreader
+from ph5 import LOGGING_FORMAT
```

Comment:
This shouldn't be necessary.
The logger in init already set the formatter

Reply:
@dsentinel This one is necessary for setting format to write log to file (line 227-232). If I don't setFormatter for this, the format on screen's log isn't affected, but the log file's will be different.

Reply:
Nevermind, you are creating another handler for file logging.